flutter/lib/src/sentry_flutter.dart

```diff
@@ -280,4 +272,20 @@ mixin SentryFlutter {
   static set native(SentryNativeBinding? value) => _native = value;
 
   static SentryNativeBinding? _native;
+
+  static Future<void> nativeCrash() {
```

Comment:
pls add docs here that this its purpose is to trigger a native crash for debug purposes etc..